### ISSUE TITLE
Transcription task: previous lines can't be cancelled with Esc

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#3314]
## Package
lib-classifier

## Describe the bug
If I click on a previously transcribed line by mistake, `Esc` doesn't dismiss the text task popup.

## To Reproduce
1. Go to a transcription workflow eg. https://www.zooniverse.org/projects/pmlogan/poets-and-lovers/classify/workflow/21362/subject-set/104803
2. Click on a pink previously transcribed line to open the subtask.
3. Press Esc to dismiss the popup. The 'Are you sure?' confirmation appears and can't be dismissed from the keyboard.

## Expected behavior
If a volunteer presses `Esc`, I think we should be okay to dismiss the subtask without asking for confimation. Even if we do ask for confirmation, it should be possible to dismiss the confimation with a single keypress eg. pressing `Esc` for a second time.

We shouldn't expect a keyboard user to have to use a mouse or trackpad to confirm a decision.
